angularSTEP/src/app/app.module.ts

```diff
@@ -4,15 +4,29 @@ import { NgModule } from '@angular/core';
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { HttpClientModule } from '@angular/common/http';
+import { MatMenuModule } from '@angular/material/menu';
+import { MatIconModule } from '@angular/material/icon';
+import { MatButtonModule } from '@angular/material/button';
+import { MatCardModule } from '@angular/material/card';
+import {MatRippleModule} from '@angular/material/core';
```

Comment:
I don't know all the style conventions yet, but for the sake of being consistent there should be a space after the open brace and before the close brace

Reply:
+1

Reply:
Thanks for pointing this out. The style issues got fixed once we implemented the Google Style Guides.